### ISSUE TITLE
Installation and setup are not clear

### ISSUE THREAD

**Issue by magom001** [matepek/vscode-catch2-test-adapter#44]
I suggest an example of how to install and setup a project is provided in the readme.

**Comment by matepek** [comment]
Hi,
I'm not sure, I understand your problem. 
What would you like to do?
Did you find my project in Visual Studio Code extension market?

**Comment by magom001** [comment]
Hi!
Yeah, I was trying to switch from clion to vs code, but never managed to get my catch test suites to be visible in the explorer. I did not find readme very helpful. A short walkthrough on how to setup a c++ project in vs code with catch would simplify things a ton!

**Comment by matepek** [comment]
I assume you've installed my [extension](https://marketplace.visualstudio.com/items?itemName=matepek.vscode-catch2-test-adapter).

1. Open Settings (-> Workspace settings)
2. Search for `catch2 executables`
3. Click `Edit in settings.json`

Now hopefully you will see the default setting, something like this:
```
// The location of your test executables (relative to the workspace folder or absolute path) and with a lot of other setting. (https://code.visualstudio.com/docs/editor/codebasics#_advanced-search-options)
  "catch2TestExplorer.executables": [
    {
      "name": "${filename} (${relDirpath}/)",
      "pattern": "{build,Build,BUILD,out,Out,OUT}/**/*{test,Test,TEST}*",
      "cwd": "${absDirpath}",
      "env": {
        "C2TESTEXECUTABLEPATH": "${absPath}"
      }
    }
  ]
```
You have to copy and personalise `catch2TestExplorer.executables` variable. (`Place your settings to the right hand side editor to override.`)
Check [README](https://github.com/matepek/vscode-catch2-test-adapter#catch2testexplorerexecutables) for personalisation details.

For deeper understanding of vscode settings click here: https://code.visualstudio.com/docs/getstarted/settings

Long story short:
Create a `.vscode/settings.json` file with customized content like this:
```
{
  "catch2TestExplorer.executables": [
    {
      "name": "${filename} (${relDirpath}/)",
      "pattern": "{build,Build,BUILD,out,Out,OUT}/**/*{test,Test,TEST}*",
      "cwd": "${absDirpath}",
      "env": {
        "C2TESTEXECUTABLEPATH": "${absPath}"
      }
    }
  ]
}
```
